timeless/access_control/authorization.py

```diff
@@ -29,4 +28,5 @@ def is_allowed(method=None, resource=None, *args, **kwargs) -> bool:
     "director": director_privileges,
     "administrator": administrator_privileges,
     "other": other_privileges,
+    "unknown": unknown_privileges
```

Comment:
@paulodamaso What is difference between `other` and `unknown`? May be we should move `unknown_privileges` to `other` ? WDYT?

Reply:
@vladarefiev Seems to me that `unknown` does have no permissions on the system, while `other` have some minor permissions (like editing own data). I've just puts these `unknown` role because tha ticket explicitly said to do so.

Reply:
@paulodamaso ok, it's also good to me